### PR TITLE
Fix issues in 'Delegated access control' section

### PR DESCRIPTION
**Pull request by sherenewso2**
## Purpose
- Fix broken links
- Fix toml configs
- Resize images
- Fix formatting

### CODE REVIEW

File: en/docs/learn/configuring-access-control-policy-for-a-service-provider.md [hunk 1]
```diff
@@ -158,8 +158,14 @@ our requirement.
...
-        ``` java
-        <Description>This policy authorizes employees of the finance team to the travelocity service provider in the authentication flow based on the roles of the user (defined by ROLE_1 and ROLE_2). Users who have at least one of the given roles, will be allowed and any others will be denied.</Description>
+        ``` xml
+        <Description>
+		This policy authorizes employees of the finance team to the travelocity 
+		service provider in the authentication flow based on the roles of 
+		the user (defined by ROLE_1 and ROLE_2). 
+		Users who have at least one of the given roles, 
+		will be allowed and any others will be denied.
+		</Description>
```
Comment (gomathyk) [inline en/docs/learn/configuring-access-control-policy-for-a-service-provider.md]: please fix indentation.

File: en/docs/learn/extension-points-for-oauth.md [hunk 2]
```diff
@@ -8,12 +8,15 @@ points related to OAuth and OpenID Connect (OIDC).
...
-place it in the *\<IS\_HOME\>/repository/component/lib* directory.  
-Then configure your extension in
-*<IS\_HOME\>/repository/conf/deployment.toml* file referring to the ' *Configuration* ' section of the
-respective extension point below.  
-Restart the server to effect changes.
+
+1. When the implementation is done, package your classes as a .jar file and
+place it in the `<IS_HOME>/repository/component/lib` directory.  
+
+2. Next, configure your extension in
```
Comment (gomathyk) [inline en/docs/learn/extension-points-for-oauth.md]: extension in the

File: en/docs/learn/extension-points-for-oauth.md [hunk 3]
```diff
@@ -229,28 +232,31 @@ If a supported grant type is to be modified you may implement a grant validator
...
-<p>!!! tip</p>
-    <p>OAuth2 token encryption protects OAuth2 access tokens, refresh tokens, consumer secrets, and authorization codes. To enable OAuth token encryption, configure the <code>                 TokenPersistenceProcessor          </code> as shown below in the <code>                 &lt;IS_HOME&gt;/repository/conf/deployment.toml        </code> file:</p>
+<pre><code>org.wso2.carbon.identity.oauth.tokenprocessor.EncryptionDecryptionPersistenceProcessor</code></pre>
+<p>Use this if you want to store access tokens, refresh tokens, authorization codes, and client secrets in an encrypted format using the OEAP (RSA/ECB/OAEPwithSHA1andMGF1Padding) algorithm. With this implementation you can use a valid token until it is either revoked or becomes invalid.
+<div class="admonition tip">
+<p class="admonition-title">Tip</p>
+    <p>OAuth2 token encryption protects OAuth2 access tokens, refresh tokens, consumer secrets, and authorization codes. To enable OAuth token encryption, configure the <code>                 TokenPersistenceProcessor          </code> as shown below in the <code>                 &lt;IS_HOME&gt;/repository/conf/deployment.toml        </code> file:
```
Comment (gomathyk) [inline en/docs/learn/extension-points-for-oauth.md]: Shall we mention TokenPersistenceProcessor as token_persistance_processor?

File: en/docs/learn/extension-points-for-oauth.md [hunk 4]
```diff
@@ -229,28 +232,31 @@ If a supported grant type is to be modified you may implement a grant validator
...
     </div>
     </div>
-    <p>Once you configure the <code>                 TokenPersistenceProcessor                </code> in the <code>  
-                   deployment.toml               </code> file, and restart WSO2 Identity Server, RSA encryption with 
-                   <strong>RSA/ECB/OAEPwithSHA1andMGF1Padding</strong> cipher transformation will be enabled by default.</p>
-<li><p><code>                 org.wso2.carbon.identity.oauth.tokenprocessor.HashingPersistenceProcessor                </code><br />
-Use this if you want to store access tokens, refresh tokens, authorization codes and client secrets as hashed values based on a define hashing algorithm. This implementation provides extra protection to vulnerabilities because a new access token is issued for every access token request. If necessary you can also use a valid token until it is either revoked or becomes invalid.<br />
+    </div>
+    </div>
+<p>Once you configure the <code> TokenPersistenceProcessor</code> in the <code>
```
Comment (gomathyk) [inline en/docs/learn/extension-points-for-oauth.md]: shall we give this also in the toml format?

File: en/docs/learn/extension-points-for-oauth.md (hunk 2 above)
Comment (sherenewso2) [reply en/docs/learn/extension-points-for-oauth.md]: Fixed in 6cad0bf12e3eca4ca70d9dea7b04128ea0b02680

File: en/docs/learn/extension-points-for-oauth.md (hunk 3 above)
Comment (sherenewso2) [reply en/docs/learn/extension-points-for-oauth.md]: Fixed in 6cad0bf12e3eca4ca70d9dea7b04128ea0b02680

**Pull request merged**